### ISSUE TITLE
Wrong Ansible Galaxy content

### ISSUE THREAD

**Issue by TawR1024** [netbox-community/ansible_modules#107]
##### ISSUE TYPE

 - Bug Report

##### SUMMARY

Ansible galaxy contains old version of code. It refers to `_to_slug` method in `netbox_utils`.
##### STEPS TO REPRODUCE

ansible-galaxy collection install or downloading tar ball from https://galaxy.ansible.com/netbox_community/ansible_modules

##### EXPECTED RESULTS

```python
def _to_slug(self, value):
        """
        :returns slug (str): Slugified value
        :params value (str): Value that needs to be changed to slug format
        """
        if value is None:
            return value
        elif isinstance(value, int):
            return value
        else:
            removed_chars = re.sub(r"[^\-\.\w\s]", "", value)
            convert_chars = re.sub(r"[\-\.\s]+", "-", removed_chars)
            return convert_chars.strip().lower()
```
##### ACTUAL RESULTS

```python
def _to_slug(self, value):
        """
        :returns slug (str): Slugified value
        :params value (str): Value that needs to be changed to slug format
        """
        if value is None:
            return value
        elif isinstance(value, int):
            return value
        elif " " in value:
            slug = value.replace(" ", "-").lower()
        else:
            slug = value.lower()
        return slug
```

**Comment by FragmentedPacket** [comment]
I haven't released `v0.1.7` that has this fix in it as there are quite a few changes I wanted to incorporate into the next release. I will try and get a new version out this weekend. I may just end up pushing what is in `devel` (the slug fixes) and the additional commits just go into a different version since we can iterate fast anyways.

**Comment by FragmentedPacket** [comment]
A new version has been pushed up to Galaxy and should match the current `develop` branch.
